data/data/powervs/iaas/power-iaas.tf

```diff
@@ -0,0 +1,22 @@
+provider "ibm" {
+ ibmcloud_api_key = var.ibmcloud_api_key
+}
+
+data "ibm_resource_group" "group" { 
+  name = var.cloud_resource_group
+}
+
+resource "ibm_resource_instance" "resource_instance" { 
+  name     = "${var.cluster_id}-power-iaas" 
+  service  = "power-iaas"
+  plan     = "power-virtual-server-group" 
+  location = var.ibmcloud_region
+  tags     = concat( var.service_tags, [ "${var.cluster_id}-power-iaas" ] )
```

Comment:
The tag should also contain `var.cluster_id` as this will be used across all resources.

Reply:
I quickly looked at aws and didn't immediately find where var.cluster_id alone was added to the list of tags, but went ahead and tagged this one with var.cluster_id anyway.